### DISCUSSION TITLE
articles/how-to-speak-by-patrick-winston

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#145] [Comments]
https://hutusi.com/articles/how-to-speak-by-patrick-winston 

 Patrick Winston1 是 AI 领域的先驱，生前曾一直担任 MIT CSAIL（计算机科学与人工智能实验室）主任。他的课程以易于理解及信息丰富而闻名，因而，他也在 MIT 开设了一个 如何演讲 “How To Speak” 的课程。2 Patrick Winston 个人主页 ↩ MIT 公开课 How To Speak ↩

**Comment by petite-dd** [comment]
小建议加个css看起来更舒服一些，
.post p {
    line-height: 2.5;
    text-indent: 2em;
}

**Comment by hutusi** [comment]
> @petite-dd
> 小建议加个css看起来更舒服一些，
> .post p {
>     line-height: 2.5;
>     text-indent: 2em;
> }

谢谢，加大行距和缩进可能确实舒服些，我对前端和设计不太擅长，回头优化下。
